server/games/game_rater.py

```diff
@@ -44,18 +44,22 @@ def _get_rating_groups(self) -> List[Dict[Player, Rating]]:
         example output: [ {p1: Rating, p2: Rating}, {p3: Rating, p4: Rating} ]
         """
         if FFA_TEAM in self._players_by_team:
+            number_of_players = sum([
+                len(self._players_by_team[team])
+                for team in self._players_by_team
+            ])
             if (
                 len(self._players_by_team[FFA_TEAM]) == 2
                 and len(self._players_by_team) == 1
             ):
                 return [{
                     player: Rating(*player.ratings[self._rating_type])
                 } for player in self._players_by_team[FFA_TEAM]]
-            else:
+            elif number_of_players != 2:
                 raise GameRatingError(
                     f"Attempted to rate FFA game with other than two players: {{team: players}} = {self._players_by_team}"
                 )
-        elif len(self._players_by_team) == 2:
+        if len(self._players_by_team) == 2:
```

Comment:
I might add a newline above, to make it clear that this is a new `if` block now.